corehq/apps/reminders/forms.py

```diff
@@ -1581,7 +1598,12 @@ def _clean_timeouts(self, value):
         return []
 
     def clean_global_timeouts(self):
-        return self._clean_timeouts(self.cleaned_data['global_timeouts'])
+        method = self.cleaned_data['method']
+        if (self.ui_type == UI_SIMPLE_FIXED and
+            method in (METHOD_SMS_CALLBACK, METHOD_SMS_SURVEY, METHOD_IVR_SURVEY)):
```

Comment:
StackOverflow discussion here: http://stackoverflow.com/a/181557

My personal preference:

    if (
        ham and
        eggs
    ):
        do_spam()